src/spark_manager.py

```diff
@@ -155,7 +155,10 @@ def create_cluster(
         self._create_master_deployment(master_cores, master_memory, postgres_env)
         self._create_master_service()
 
-        # TODO: Create the Spark worker deployment
+        # Create the Spark worker deployment
+        self._create_worker_deployment(
```

Comment:
Can you remind me where `create_cluster` gets called from

Reply:
It will be an API call. At the very beginning of each user's Jupyterhub session, we should call this method via an API call and create the user specific spark cluster. For some power users (I haven't built the auth part yet), they should be able to call this API and create their own spark clusterings.

Reply:
Ok for now it's not called anywhere in the codebase yet?

Reply:
not yet. I will need to build the API part first then the KBase auth part before make this effective.